actors/builtin/multisig/multisig_actor.go

```diff
@@ -141,6 +146,9 @@ func (a Actor) Propose(rt vmr.Runtime, params *ProposeParams) *ProposeReturn {
 	rt.ValidateImmediateCallerType(builtin.CallerTypesSignable...)
 	callerAddr := rt.Message().Caller()
 
+	resolvedCaller, err := resolveToIDAddr(callerAddr, rt.ResolveAddress, rt.Send)
+	builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to resolved address %v", callerAddr)
```

Comment:
Unnecessary, the caller is always an ID address.

Reply:
This is fixed everywhere.